A79-non-per-call-metrics-architecture.md

```diff
@@ -0,0 +1,405 @@
+    kCounter,
+    kHistogram,
+  };
+  struct GlobalInstrumentDescriptor {
+    ValueType value_type;
+    InstrumentType instrument_type;
+    uint32_t index;
+    bool enable_by_default;
+    absl::string_view name;
+    absl::string_view description;
+    absl::string_view unit;
+    std::vector<absl::string_view> label_keys;
+    std::vector<absl::string_view> optional_label_keys;
+  };
+  struct GlobalInstrumentHandle {
+    uint32_t index;
+  };
+  struct GlobalUInt64CounterHandle : public GlobalInstrumentHandle {};
+  struct GlobalDoubleHistogramHandle : public GlobalInstrumentHandle {};
+
+  // Creates instrument in the GlobalInstrumentsRegistry.
+  static GlobalUInt64CounterHandle RegisterUInt64Counter(
+      absl::string_view name, absl::string_view description,
+      absl::string_view unit, absl::Span<const absl::string_view> label_keys,
+      absl::Span<const absl::string_view> optional_label_keys,
+      bool enable_by_default);
+  static GlobalDoubleHistogramHandle RegisterDoubleHistogram(
+      absl::string_view name, absl::string_view description,
+      absl::string_view unit, absl::Span<const absl::string_view> label_keys,
+      absl::Span<const absl::string_view> optional_label_keys,
+      bool enable_by_default);
+  static void ForEach(
+      absl::FunctionRef<void(const GlobalInstrumentDescriptor&)> f);
+};
+```
+
+##### Java
+
+(To be filled.)
+
+##### Go
+
+(To be filled.)
+
+#### Global Stats Plugin Registry
+
+A global stats plugin registry is added. This registry keeps track of all stats
+plugins that have been registered globally.
+
+The application will use a stats plugin builder (e.g.
+`OpenTelemetryPluginBuilder`) to register a stats plugin to the registry. On
+creation, stats plugins should query the global instruments registry to get the
+list of instrument descriptors. Based on these descriptors and the stats
+plugin's configuration, backing instruments (e.g., `OpenTelemetry Instruments`)
+will be created.
+
+![](A79_graphics/global-stats-plugin-registry-usage.png)
+
+##### Scoping Stats Plugins for Channels
+
+To allow stats plugins to scope themselves to a certain set of channels,
+implementations can choose one of two approaches.
+
+In the first approach, stats plugins provide a scope filter that applications
+can configure. Based on the channel's configuration, a stats plugin can decide
+whether it wants to record metrics for the channel. At channel creation time,
+the channel's target and the authority is provided to the stats plugin to make
+this decision.
+
+![](A79_graphics/stats-plugin-scoping.png)
+
+In the second approach, stats plugins that are to be scoped to a channel are
+registered with the corresponding channel directly, instead of the global stats
+plugin registry. On channel creation time, stats plugins from the registry are
+combined with the list of stats plugins registered directly on the channel to
+form the complete list of stats plugins for this channel. There is a known issue
```

Comment:
I would start a new paragraph with the sentence starting with "There is a known issue", since this is really a separate concept from what the first couple of sentences are talking about.

Reply:
done